apps/playground-vite/package.json

```
@@ -0,0 +1,49 @@
+{
+  "scripts": {
+    "dev": "pnpm run server",
+    "prod": "pnpm run build && pnpm run server:prod",
+    "build": "vite build",
+    "server": "tsx ./server",
+    "server:prod": "cross-env NODE_ENV=production tsx ./server"
+  },
+  "dependencies": {
+    "@vitejs/plugin-react": "^2.2.0",
+    "body-parser": "^1.20.1",
+    "cookie-parser": "^1.4.6",
+    "cross-env": "^7.0.3",
+    "dotenv": "^10.0.0",
+    "express": "^4.18.1",
+    "next-auth": "^4.16.4",
```

```suggestion
    "next-auth": "worskspace:*",
```

Might need to rerun `pnpm i`